data-reconciliation-app/Makefile

```diff
@@ -9,7 +9,6 @@ help: ## 💬 This help message :)
 build: ## 🔨 Build the Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
-	@npm run create-jwt-config # Create the JWT issuer config files for (Test - Microsoft Azure Identity Provider).
```

Comment:
Removed this to avoid intermittent failure on this patch.
The same change is done in #217 
Will be resolved on merge